bank_atm.rb

```diff
@@ -0,0 +1,96 @@
+$LOAD_PATH << '.'
+require 'validations'
+require 'models'
+
+#Begin of Demo data
+p1 = Person.new("John", "Developer", "j@yahoo.com")
+p2 = Person.new("Tudor", "Climbing instructor", "t@tahoo.com")
+
+persons = []
+persons << p1
+persons << p2
+
+b1 = BankAccount.new("1234", p1, 150)
+b2 = BankAccount.new("5555", p1, 5500)# with 1 widthrawal already
+b3 = BankAccount.new("9999", p2, 7000)
+
+b2.withdraw(4000)
+#End of demo data
+
+#Start of the flow
+loop do
+    puts "\nWelcome to the ATM!"
+    puts "Select an option: "
+    #Options
+    puts "1. Deposit funds"
+    puts "2. Withdraw funds"
+    puts "3. Create account"
+    puts "4. View all accounts"
+    puts "5. Quit"
+
+    option = gets.chomp.downcase
+    break if option == "5"
+    puts "You selected #{option}"
+
+    case option
```

Comment:
I think a better option here would be to extract the logic into separate methods:
```ruby
case option
when 1
  deposit_funds
when 2
  withdraw_funds
when 3
  create_account
# ... etc
```

This way, the logic can be found in methods, easier to find them and the switch won't get very long